config/core.extension.yml

```diff
@@ -15,6 +15,7 @@ module:
   ctools: 0
   datetime: 0
   datetime_range: 0
+  devel: 0
```

Comment:
Devel module is enabled via config_split: https://github.com/UN-OCHA/rwint9-site/blob/c41fd92c2e6ecb511bbb0001b616099d63a232ac/config/config_split.config_split.config_dev.yml#L14